templates/includes/record-revealed.html

```diff
@@ -7,10 +7,10 @@
                 <h1 class="explore-intro__title">{{ page.title }}</h1>
                 <p class="explore-intro__paragraph">{{ page.intro|richtext }}</p>
             </div>
-            <div class="col-4 explore-intro__image">
+            <div class="col-4 explore-intro__image-container">
                 {% if page.teaser_image %}
                     {% image page.teaser_image fill-390x390 as teaser_image %}
-                    <img height=390 width=390 src="{{ teaser_image.url }}" alt="" />
+                    <img height=390 width=390 src="{{ teaser_image.url }}" alt="" class="explore-intro__image" />
                 {% endif %}
```

Comment:
This makes the intro image a little more flexible so it's not causing layout shift.